### PR TITLE
Add new skill to provide the ability to extract the main questions from e-mails

### PR DESCRIPTION
**Pull request by celestelgarrido**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- This skill provides the ability to extract the main questions from e-mails for later use in semantic searches or other actions.

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: compositional_skills/extraction/email/main_questions/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 1
+task_description: |
+    This skill provides the ability to extract the main questions from e-mails for later use in semantic searches or other actions.
+created_by: celestelgarrido
+seed_examples:
+  - context: |
+      I'm encountering a software bug in the Microsoft Surface?. Whenever I try to perform a specific action, the application crashes. Are there any updates or fixes available? On Windows Vista, this is not possible. If you are I've performed a factory reset on my Microsoft Surface?, hoping it would resolve the problem, but it didn't help.
```
[the remark this answers is not in the document]
Comment (celestelgarrido) [reply compositional_skills/extraction/email/main_questions/qna.yaml]: Changed

File: compositional_skills/extraction/email/main_questions/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    question: |
+      could you extract a list of the main questions from the following email?
+    answer: |
+      - Are there any updates or fixes available for the software bug in the Microsoft Surface?
+      - Is it possible to perform a specific action on Windows Vista without the application crashing?
+      - Did performing a factory reset on the Microsoft Surface help resolve the issue?
+  - context: |
+      I've recently set up my GoPro Hero, but it fails to connect to any available networks. What steps should I take to troubleshoot this issue? If you know the answer, please let me know. Do I I've noticed a sudden decrease in battery life on my GoPro Hero. It used to last much longer.
```
[the remark this answers is not in the document]
Comment (celestelgarrido) [reply compositional_skills/extraction/email/main_questions/qna.yaml]: Changed

**Review by celestelgarrido — COMMENTED** [review]
I have removed each product names

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues listed here: https://github.com/instructlab/taxonomy/pull/1155/files

**Pull request closed without merging**